srcpkgs/qmk/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'qmk'
+pkgname=qmk
+version=0.0.27
+revision=1
+archs=noarch
+create_wrksrc=yes
+build_wrksrc="./qmk-${version}"
```

Comment:
Are `{create,build}_wrksrc` really needed? Should work without them.

Reply:
In this case it is necessary the wrksrc because when uncompressed the distfiles create the qmk-${version} where the setup.py resides so I had to create it to cd into the right folder.

Reply:
Yes, but wrksrc defaults to `$pkgname-$version`, so `qmk-$version`. This looks like smth that just wasn't removed when renaming the template from `python3-qmk` to `qmk`

Reply:
Let me try with it.

Reply:
It still doesn't work, it creates a subfolder even with the name change.

Reply:
Yes, you were right, finally got it

Reply:
you should drop *both* create_wrksrc and build_wrksrc

Reply:
Right, messed up really bad. Sorry once more pff.